tests/testthat/test_comparePlot.R

```diff
@@ -61,6 +54,40 @@ test_that("comparePlot.R: comparePlot()", {
     expect_setequal(unlist(cPlot$labels[c("title","y", "x")]), c("MyPlot", "yaxis-title", "xaxis-title"))
     expect_setequal(cPlot$layers[[2]]$aes_params$colour, "grey50")
 
+    # testing aesthetics of plots without significance measures
+    cPlot <- comparePlot(compareDat[, 1:2],
+                         pThreshold = 0.001,
+                         title    = "MyPlot",
+                         xlab     = "xaxis-title",
+                         ylab     = "yaxis-title",
+                         symbolSize = c(5, 5, 2, 2),
+                         symbolShape = c(21, 21, 21, 20),
+                         symbolFill = c("red", "blue", "yellow", "grey25"),
+                         alpha = 0.5,
+                         crosshair = "grey50",
+                         referenceLine = "darkgoldenrod1",
+                         refLineThickness = 1,
+                         legendPosition = "right",
+                         footnote = "This is my footnote")
+    expect_s3_class(cPlot , c("canvasXpress", "htmlwidget"))
+    cPlot <- comparePlot(compareDat[, 1:2],
+                         plotType = "ggplot",
+                         pThreshold = 0.001,
+                         title    = "MyPlot",
+                         xlab     = "xaxis-title",
+                         ylab     = "yaxis-title",
+                         symbolSize = c(5, 5, 2, 2),
+                         symbolShape = c(21, 21, 21, 20),
+                         symbolFill = c("red", "blue", "yellow", "grey25"),
+                         alpha = 0.5,
+                         crosshair = "grey50",
+                         referenceLine = "darkgoldenrod1",
+                         refLineThickness = 1,
+                         legendPosition = "right",
+                         footnote = "This is my footnote")
+    expect_setequal(unlist(cPlot$labels[c("title","y", "x")]), c("MyPlot", "yaxis-title", "xaxis-title"))
+    expect_setequal(cPlot$layers[[2]]$aes_params$colour, "grey50")
+
```

Comment:
Test cases for optional parameters need to be added.

Reply:
done